lib/elixir/lib/calendar/iso.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Calendar.ISO do
```

Comment:
@bitwalker at this point I am almost confident we will ship with a single calendar implementation in Elixir and leave the remaining ones as extension points. So while we wouldn't merge others calendars, your knowledge on supporting multiple calendars will be extremely important.